src/impl/tpls/KokkosBlas1_scal_tpl_spec_decl.hpp

```diff
@@ -88,6 +88,7 @@ Kokkos::View<const double*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
   \
   static void scal (const RV& R, const double& alpha, const XV& X) \
   { \
+    Kokkos::Profiling::pushRegion("KokkosBlas::scal[TPL_BLAS,double]"); \
```

Comment:
@kyungjoo-kim I am just curious about profiling. I have seen you put ```pushRegions``` and ```popRegions``` around these function calls. Is it adding a lot overheads?

Reply:
@vqd8a those are added profiling hooks for using the kokkos-tools. The cost is a null-op unless you are using the tools.